examples/dwarfdump.rs

```diff
@@ -491,7 +494,7 @@ where
         let data = match file.section_by_name(id.name()) {
             Some(ref section) => {
                 add_relocations(&mut relocations, file, section);
-                section.uncompressed_data()
+                section.uncompressed_data().unwrap()
```

Comment:
This should return an error instead of unwrapping.

Reply:
Now returning the error, please check